docs/client-libraries-tableviews.md

```diff
@@ -4,36 +4,106 @@ title: Work with TableView
 sidebar_label: "Work with TableView"
 ---
 
+````mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+````
+
 After setting up your clients, you can explore more to start working with [TableView](concepts-clients.md#tableview).
 
 ## Configure TableView
 
-The following is an example of how to configure a TableView.
 
-```java
-TableView<String> tv = client.newTableViewBuilder(Schema.STRING)
-  .topic("my-tableview")
-  .create()
-```
+````mdx-code-block
+<Tabs groupId="lang-choice"
+defaultValue="Java"
+values={[{"label":"Java","value":"Java"},{"label":"C++","value":"C++"}]}>
+<TabItem value="Java">
+
+  The following is an example of how to configure a TableView.
+  
+  ```java
+    TableView<String> tv = client.newTableViewBuilder(Schema.STRING)
+    .topic("my-tableview")
+    .create()
+  ```
+
+You can use the available parameters in the `loadConf` configuration or the API [`TableViewBuilder`](/api/client/@pulsar:version_number@/org/apache/pulsar/client/api/TableViewBuilder.html) to customize your TableView.
```

Comment:
Have you previewed the API link?

Reply:
oh, sorry. this link is wrong.

<img width="1299" alt="image" src="https://github.com/apache/pulsar-site/assets/33416836/40f9f32d-8c4d-4b16-abe3-3cd78c681fcc">

@tisonkun Can you help take a look?

The original thread is here
https://github.com/apache/pulsar-site/pull/565#discussion_r1189257265

Reply:
```suggestion
You can use the available parameters in the `loadConf` configuration or the API [`TableViewBuilder`](/api/client/org/apache/pulsar/client/api/TableViewBuilder.html) to customize your TableView.
```

The replace logic can be found at:

https://github.com/apache/pulsar-site/blob/22081daf8095676f897a0224351e6116d7d0d183/scripts/replace.js#L199

You can run `node scripts/replace.js` to test locally.

Reply:
Thanks, I test it, and it can work.